gpytorch/distributions/multivariate_normal.py

```diff
@@ -180,10 +180,9 @@ def sample(self, sample_shape=torch.Size(), base_samples=None):
     def variance(self):
         if self.islazy:
             # overwrite this since torch MVN uses unbroadcasted_scale_tril for this
-            if len(self._batch_shape) == 2:
-                return self.lazy_covariance_matrix.diag().unsqueeze(-1).expand(self._batch_shape + self._event_shape)
-            else:
-                return self.lazy_covariance_matrix.diag().expand(self._batch_shape + self._event_shape)
+            diag = self.lazy_covariance_matrix.diag()
+            diag = diag.view(*diag.shape[:-1], *self._event_shape)
```

Comment:
we can do that as the convention.

Reply:
For what it's worth I prefer comma separated because `mat.view(*some_shape, 1, *other_shape)` works but you'd have to do `mat.view(some_shape + torch.Size([1]) + other_shape)` with `+`.

Reply:
Yeah I'd use `mat.view(*some_shape, 1, *other_shape)` in that case. I don't have very strong feelings about this...